jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -28,7 +28,13 @@
     - workspace-cleanup:
         dirmatch: true
         exclude:
-        - 'test-infra/.git/'
+        - '**/.git/'
+        external-deletion-command: 'sudo rm -rf %s'
+    publishers:
```

Comment:
Will you add a comment explaining a) how these two workspace-cleanup entries differ and b) why we need both of them?